homework7part1.py

```diff
@@ -0,0 +1,59 @@
+phone_list = [
+    ["Robb", "0680929258"],
+    ["Stark", "380912146146"],
+    ["Jonn", "+380636690103"],
+    ["Snow", "+380638690261"],
+    ["Tirion", "+958758767"]
+]
+
+phone_dict = dict(phone_list)
+
+print(
+    'Список команд які можна використати в даній телефонній книзі:',
+    "Формат: команда, номер, ім'я",
+    '- stats: кількість записів',
+    '- add: додати запис',
+    '- del: видалити запис за іменем (ключем)',
+    '- list: список всіх імен в книзі',
+    '- show: детальна інформація по імені',
+    sep='\n'
+)
+
+while True:
+    command = input('Enter a command: ')
+# Переглянути кількість записів
+    if command == 'stats':
+        print(len(phone_dict))
+# Додати нового користувача
+
+    elif command == 'add':
+        while True:
```

Comment:
There is not much sense in this loop since you always have here only one iteration